src/icesat2_tracks/local_modules/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.0"
```

Comment:
This also doesn't need a \_\_version\_\_
```suggestion
"""Plotting tools for the icesat2_tracks package"""
```